docs/actions/actions.md

```diff
@@ -0,0 +1,23 @@
+# Actions
+
+Plone has the concept of configurable actions called `portal_actions`.
+Each action defines an `id`, a `title`, the required permissions, and a condition that will be checked to decide whether the action will be available for a user.
+Actions are sorted by categories.
+
+Actions can be used to build UI elements that adapt to the available actions.
```

Comment:
```suggestion
Actions can be used to build user interface elements that adapt to the available actions.
```

Reply:
Hi @stevepiercy I had a discussion with Victor prior to writing the documentation. I was thinking of going with JSDoc, but based on Victor's recommendations, I opted for writing the documentation in MyST syntax. The documentation is mostly complete. Not entirely sure if it's worth doing a complete rewrite, but I'll definitely take a closer look.

Reply:
I figured something like that happened. I don't know whether you and @sneridagh knew about https://github.com/mozilla/sphinx-js when you had that discussion.

Here's a "juicy example" of the output, as Mozilla describes it in their readme:

https://mozilla.github.io/fathom/ruleset.html

If you don't use JSDoc comments in your API code, then (1) I think that is unfortunate, and (2) you won't be able to automatically generate documentation. I think it makes code and documentation more maintainable because it's all in one place to update, you don't have to manually format all the bits and pieces, the output is really nice. As the Plone documentation lead cat herder, I think it is worth doing, but I have a strong bias for maintainable (especially from automation), well-structured, beautiful docs.

That said, it might not be a good fit for this project. It's up to @sneridagh to decide, or maybe create a new issue for a new contributor to work on in the future.